src/lib/spacefinder/liveblog-adverts.ts

```diff
@@ -119,17 +114,15 @@ const insertAdAtPara = (para: Node): Promise<void> => {
 					adSizes.outstreamDesktop,
 					adSizes.outstreamGoogleDesktop,
 				],
-			});
-			if (advert) {
-				insertedDynamicAds.push(advert);
-			}
-		});
+			}),
+		);
 };
 
 const insertAds: SpacefinderWriter = async (paras) => {
 	const fastdomPromises = [];
 	for (let i = 0; i < paras.length && AD_COUNTER < MAX_ADS; i += 1) {
 		const para = paras[i];
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- false positive
```

Comment:
You might be able to remove these comments if you rebase off the latest main, which could require passing `--no-verify` to `git push` if the validation is failing. Happy for you to leave them if that's too much of a fuss though.